rd_stats.c

```diff
@@ -1572,6 +1572,22 @@ __nr_t read_net_eicmp(struct stats_net_eicmp *st_net_eicmp)
 				break;
 			}
 			else {
+				if (!strlen(format)) {
+					if (strstr(line, "InCsumErrors")) {
+						/*
+						 * New format: InCsumErrors field exists at position #3.
+						 */
+						strcpy(format, "%*u %*u %lu %lu %lu %lu %lu %lu %*u %*u "
```

Comment:
Should be `strcpy(format, "%*u %lu %*u %lu..."`

Reply:
Indeed. Good catch!